specs/altair/light-client/sync-protocol.md

```diff
@@ -150,6 +152,20 @@ class LightClientStore(object):
 
 ## Helper functions
 
+### `get_lc_beacon_slot`
```

Comment:
Related discussion: https://github.com/ethereum/consensus-specs/pull/3151#pullrequestreview-1236859997

Whether a `LightClientHeader` wrapper / typedef should be introduced for `BeaconBlockHeader` for consistency with following forks.